meson_options.txt

```diff
@@ -16,6 +16,12 @@ option(
   description : 'generate man pages',
   value : 'auto',
 )
+option(
+  'tests',
+  type : 'feature',
+  description : 'build tests',
+  value : 'enabled',
```

Comment:
```suggestion
  type : 'boolean',
  description : 'build tests',
  value : true,
```

Reply:
Thanks for your review, I updated the PR